pkg/asset/installconfig/ovirt/validation.go

```diff
@@ -125,3 +126,22 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+// ValidateProvisioning - Perform Provisioning Validations for oVirt
+func ValidateProvisioning(ic *types.InstallConfig) error {
+	clusterName := ic.ObjectMeta.Name
+	zoneName := ic.BaseDomain
```

Comment:
there is already a cluster domain function on the install config for this AFAIR

Reply:
not sure what you mean.. this function invoked on  ovirt validation only